backend/src/case_file/case_file_inputs.graphql

```diff
@@ -200,6 +200,20 @@ input DrugInput {
   order: Int
 }
 
+input DrugInputV2 {
+  id: String
+
+  vial: String!
+  drug: String!
+  amountUsed: String!
+  injectionMethod: String!
+
+  remainingUse: String
+  additionalComments: String
+
+  order: Int
+}
+
 input WildlifeActionInput {
```

Comment:
As we are currently the only consumer of the API, do we need to version the input?

Reply:
Corrected this with ea9d7b3.